cloudpickle/compat.py

```diff
@@ -7,7 +7,7 @@
         from pickle5 import Pickler  # noqa: F401
     except ImportError:
         import pickle  # noqa: F401
-        from pickle import _Pickler as Pickler  # noqa: F401
+        from pickle import Pickler as Pickler  # noqa: F401
 else:
     import pickle  # noqa: F401
-    from _pickle import Pickler  # noqa: F401
+    from pickle import Pickler  # noqa: F401
```

Comment:
This change on the other hand should be fine: on recent Python versions `from pickle import Pickler` imports the fast C implementation from the `_pickle` builtin module (when available) and falls back to the C variant if Python was not built with this module.